config/routes.rb

```diff
@@ -1,12 +1,14 @@
 Rails.application.routes.draw do
 	root 'test#index'
 	get 'test/index'
-	
+
 	post 'api/signup' #=> 'api#signup'
 	post 'api/signin' #=> 'api#signin'
 
 	get 'api/get_token'
 	get 'api/clear_token'
 
+	post 'api/post_location'
```

Comment:
Jose went off LOOL

On Tue, Mar 22, 2016 at 7:58 PM, Javier Ruiz <notifications@github.com>
wrote:

> In config/routes.rb
> <https://github.com/JavierR14/GpsApp-Server/pull/1#discussion_r57090579>:
>
> >  	post 'api/signup' #=> 'api#signup'
> >  	post 'api/signin' #=> 'api#signin'
> >
> >  	get 'api/get_token'
> >  	get 'api/clear_token'
> >
> > +	post 'api/post_location'
>
> move this to where the rest of the posts for api are
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JavierR14/GpsApp-Server/pull/1/files/f425e9aee7f6ff46e01d6de8a37e208d4d5b97e2#r57090579>
>